repo/meta/schema/package-schema.json

```diff
@@ -68,6 +72,7 @@
     }
   },
```

Comment:
should we also remove `images` from this schema since they should now be in `resource.json`?